.github/workflows/quality.yaml

```diff
@@ -0,0 +1,95 @@
+name: Quality
```

Comment:
Curiosity: Why is it named `Quality`? (not `Test` or smth)

Reply:
Just copied it from somewhere. But I can rename it to `Tests`